### PR TITLE
Promote the appstore more

### PR DESCRIPTION
**Pull request by jospoortvliet**
A variety of pages gets a section promoting our app store:
* sharing page
* monitoring page
* storage page
* features page (improved existing one)

I also renamed the 'support' option to resources and added the app store there... But that one I'm not 100% sure about, input super welcome!

Screenshots:
spectacle w24045

image

### CODE REVIEW

**Comment by MariusBluem** [comment]
I like the renaming of Support to Resources, since Support is something different for me...

**Comment by jospoortvliet** [comment]
Now: TESTABLE ON STAGING! GRAB IT WHILE IT'S HOT!
nextcloud.com/sharing/?staging=true
nextcloud.com/monitoring/?staging=true
nextcloud.com/storage/?staging=true
https://nextcloud.com/webrtc/?staging=true

**Comment by jospoortvliet** [comment]
@MariusBluem any chance you can approve if you like it and see no issues ;-)

**Comment by MariusBluem** [comment]
Tests are failing - so no, until fixed 🙈

**Comment by jospoortvliet** [comment]
@MorrisJobke @LukasReschke what is 'codacy/pr' and how do I get rid of it?

File: assets/css/webrtc.scss [hunk 1]
```diff
@@ -47,4 +51,4 @@ footer {
 	.hyperlink {
 		color: white;
 	}
-}
+}*/
```
Comment (MorrisJobke) [inline assets/css/webrtc.scss]: Drop it.

File: assets/css/webrtc.css [hunk 2]
```diff
@@ -16,30 +16,38 @@
...
+	.row {
+		margin-bottom: 0px;
+	}
+	p {
+		color: white;
+	}
+	.hyperlink {
+		color: white;
+	}
+}*/
```
Comment (MorrisJobke) [inline assets/css/webrtc.css]: Drop it.

File: assets/css/storage.scss [hunk 3]
```diff
@@ -13,7 +13,7 @@
 }
 
 section {
-	margin-top: 150px;
+/* 	margin-top: 150px; */
```
Comment (MorrisJobke) [inline assets/css/storage.scss]: Drop it.

File: assets/css/storage.css [hunk 4]
```diff
@@ -16,19 +16,19 @@
   float: right; }
 
 section {
-  margin-top: 150px;
+  /* 	margin-top: 150px; */
```
Comment (MorrisJobke) [inline assets/css/storage.css]: Drop it.

File: assets/css/custom.scss [hunk 5]
```diff
@@ -32,7 +32,61 @@
...
+		}
+
+		.fa {
+			color: white;
+			padding-right: 10px;
+		}
 
+		.col-md-6 {
+			padding: 80px 0px;
+/* 			margin-left: 10% */
```
Comment (MorrisJobke) [inline assets/css/custom.scss]: Don't add it.

File: assets/css/custom.scss [hunk 6]
```diff
@@ -32,7 +32,61 @@
...
+		}
+
+		.big-img {
+			position: absolute;
+			width: 70%;
+	/*         top: 60px; */
```
Comment (MorrisJobke) [inline assets/css/custom.scss]: Don't add it.

File: assets/css/custom.css [hunk 7]
```diff
@@ -454,6 +454,44 @@ template, [hidden] {
...
+    .benefits .banner .fa {
+      color: white;
+      padding-right: 10px; }
+    .benefits .banner .col-md-6 {
+      padding: 80px 0px;
+      /* 			margin-left: 10% */ }
+    .benefits .banner .big-img {
+      position: absolute;
+      width: 70%;
+      /*         top: 60px; */
```
Comment (MorrisJobke) [inline assets/css/custom.css]: Don't add it.

File: assets/css/custom.css [hunk 8]
```diff
@@ -454,6 +454,44 @@ template, [hidden] {
...
+      color: white; }
+    .benefits .banner li {
+      text-align: left;
+      color: white; }
+    .benefits .banner .fa {
+      color: white;
+      padding-right: 10px; }
+    .benefits .banner .col-md-6 {
+      padding: 80px 0px;
+      /* 			margin-left: 10% */ }
```
Comment (MorrisJobke) [inline assets/css/custom.css]: Don't add it.

**Comment by jancborchardt** [comment]
Seems good! I’m only worried a bit that the renaming from the clear »Support« to the generic »Resources« will confuse people as to where to click to get help. :\

**Comment by MariusBluem** [comment]
@jancborchardt They will still find Support in the footer if they are **that** lost, what I don’t think. Under download, they will find links to our „resources/support“ too... the support term is for me the enterprise one - community „support“ with the more or less „self service“ is covered better with „Resources“. It makes this menu more flexible in adding stuff too - e.g. „security information“ is resources and not support ...

**Comment by jospoortvliet** [comment]
I agree with Marius, I think it is OK. Plus, at some point I can create a 'whitepapers' section and put it there, too, that fits under resources but not under support... It is slightly less clear but actually more accurate and I think it's an improvement.

If one of you could approve... the stupid check of codacy/pr I will ignore, again.

**Pull request merged**